### PR TITLE
Change config to protect body, summary.

### PR DESCRIPTION
**Pull request by BobWall23**
Change title to be unprotected.
Update README appropriately.
Update to version 0.4.0 of Cloaked Search, and change configuration
accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,10 @@ docker-compose up
 
 ## Indexing
 
-`try-cloaked-search` includes some test data. Since Cloaked Search uses a different key per (tenant, index, field), documents with protected fields must be tagged with the tenant they belong to. Half of the articles in the test dataset are associated with `tenant-1`, and the other half are not associated with any tenant. Only documents belonging to `tenant-1` will be encrypted. To better understand Cloaked Search's key management, refer to the [configuration documentation](/docs/saas-shield/cloaked-search/configuration).
+`try-cloaked-search` includes some test data. Since Cloaked Search uses a different key per (tenant, index, field),
+documents with protected fields must be tagged with the tenant to which they belong.
+Half of the articles in the test dataset are associated with `tenant-1`, and the other half are not associated with any tenant.
+Only documents belonging to `tenant-1` will be encrypted. To better understand Cloaked Search's key management, refer to the [configuration documentation](/docs/saas-shield/cloaked-search/configuration).
```
Comment (coltfred) [inline README.md:43]: This link is not correct. I think you meant to link to https://ironcorelabs.com/docs/saas-shield/cloaked-search/configuration/

File: README.md [hunk 2]
```diff
@@ -50,13 +53,16 @@ _Note that we are making this request directly to Elasticsearch (port 9200) so w
 Let's get all the documents belonging to `tenant-1` and see what's in the index!
 
 ```bash
-curl -s -G --data-urlencode "q=+tenant_id:\"tenant-1\"" localhost:9200/try_cloaked_search/_search | jq
+curl -s -G --data-urlencode "q=+tenant_id:tenant-1" localhost:9200/try_cloaked_search/_search | jq
```
Comment (coltfred) [inline README.md:56]: This change won't work once the improved tenant detection PR lands. Any tenant id which isn't entirely alphanumeric must be quoted.

File: README.md [hunk 3]
```diff
@@ -72,29 +78,41 @@ curl -s -G --data-urlencode "q=title:list" localhost:9200/try_cloaked_search/_se
 These are a couple examples of simple term queries:
 
 ```bash
-curl -s -G --data-urlencode "q=+tenant_id:\"tenant-1\" AND title:Japan" localhost:8675/try_cloaked_search/_search | jq
+curl -s -G --data-urlencode "q=+tenant_id:tenant-1 AND title:Japan" localhost:8675/try_cloaked_search/_search | jq
```
Comment (coltfred) [inline README.md:81]: Ditto on the quoting. This used to be querying a protected field, which showed that the proxy was doing its job. This is now an unprotected field. I see you expanded below to indicate that we change for querying unprotected fields to protected ones, but that isn't clear at this point in the document.

File: README.md [hunk 4]
```diff
@@ -72,29 +78,41 @@ curl -s -G --data-urlencode "q=title:list" localhost:9200/try_cloaked_search/_se
...
+
+```bash
+curl -s -G --data-urlencode "q=+tenant_id:tenant-1 AND summary:glasgow" localhost:8675/try_cloaked_search/_search | jq
 ```
 
 Term queries can also be combined with ORs or ANDs like this:
 
 ```bash
-curl -s -G --data-urlencode "q=+tenant_id:\"tenant-1\" AND (title:cup OR title:Japan)" localhost:8675/try_cloaked_search/_search | jq
+curl -s -G --data-urlencode "q=+tenant_id:tenant-1 AND (summary:cup OR title:Japan)" localhost:8675/try_cloaked_search/_search | jq
```
Comment (coltfred) [inline README.md:103]: This used to show 2 fields that are protected both being queried and now it's one protected and one plaintext. Maybe we should have both types of queries to show that they can be mixed and matched.

File: README.md [hunk 5]
```diff
@@ -72,29 +78,41 @@ curl -s -G --data-urlencode "q=title:list" localhost:9200/try_cloaked_search/_se
...
 ```bash
-curl -s -G --data-urlencode "q=+tenant_id:\"tenant-1\" AND title:\"Cheerleading in Japan\"" localhost:8675/try_cloaked_search/_search | jq
+curl -s -G --data-urlencode "q=+tenant_id:tenant-1 AND summary:\"Cheerleading in Japan\"" localhost:8675/try_cloaked_search/_search | jq
 ```
 
 Finally, here is an example of a prefix query:
 
 ```bash
-curl -s -G --data-urlencode "q=+tenant_id:\"tenant-1\" AND title:list*" localhost:8675/try_cloaked_search/_search | jq
+curl -s -G --data-urlencode "q=+tenant_id:tenant-1 AND title:list*" localhost:8675/try_cloaked_search/_search | jq
```
Comment (coltfred) [inline README.md:115]: This doesn't show prefix querying on protected fields anymore.

File: test-key [hunk 6]
```diff
@@ -0,0 +1,4 @@
+standalone_keys:
```
Comment (coltfred) [inline test-key:1]: See https://github.com/IronCoreLabs/try-cloaked-search/pull/7#issuecomment-929282290

**Review by coltfred — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (BobWall23) [reply README.md:43]: Oops - I was thinking of the website pages..

**Comment by BobWall23** [comment]
@coltfred I think I addressed everything

**Review by coltfred — APPROVED** [review]
Thanks for addressing everything, Bob!

**Pull request merged**